utils/test-utils/src/main/resources/logback-test.xml

```diff
@@ -61,5 +61,6 @@
     <!-- skuber by default logs all requests to k8s on INFO -->
     <logger name="skuber" level="${SKUBER_LOG_LEVEL:-WARN}"/>
     <logger name="com.azure" level="INFO"/>
+    <logger name="org.testcontainers" level="DEBUG"/>
```

Comment:
Added a debug logging for testcontainers. Was really useful and I think it will be useful as default.